mantidproject/mantid
# Change the progress bar to better handle multiple running algorithms

Changes the "Details" button to read "Running n" or "Idle"

Progress bar displays the progress of the most recently started algorithm, and will now revert to the next most recently started one if it finishes first.


**To test:**



1. Start Mantidplot
1. Run Pause, for 50 seconds
1. While running, run another set to 5 seconds
1. observe
1. Log onto a catalog (this will leave a catalog keep alive algorithm running)
1. run some other algorithms
1. Start a live data Load (this will leave a monitorLiveData algorithm running)
1. run some other algorithms

Fixes #18544

###  RELEASE NOTES

in ui.rst

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
